de.metas.async/src/main/java/de/metas/async/model/validator/Main.java

```diff
@@ -57,7 +57,11 @@
  */
 public class Main extends AbstractModuleInterceptor
 {
-	public static transient String SYSCONFIG_Async_InitDelayMillis = "Async_InitDelayMillis";
+	/**
```

Comment:
* i think for a dev looking at the code, it's clear. also because that constant is (afaiu) used only on this class.
  * to that end, please make the two constants `private` if they aren't intended to be used anywhere else. If they are referenced in a test case, plase annotate them with `@VisibleForTesting` from the guava library
* but i'm fine with the comment if you want to keep it
* a good constant should be not only upper-case, fut also `final` (not `transient `) :-P

Reply:
Done:
* deleted the javadoc. It's undertstandable from the code what that sys config is supposed to be, and in the database the description is kept. So if somebody wants to update it will know what it means from there.
* made the constants private. They are not used in any test.
* made the constant uppercase and final.

Thanks for the review! :)